Orange/widgets/data/tests/test_oweditdomain.py

```diff
@@ -917,6 +918,19 @@ def test_column_str_repr(self):
         d = column_str_repr(v, np.array([0., np.nan, 1.0]))
         assert_array_equal(d, ["00:00:00", "?", "00:00:01"])
 
+    def test_time_parse(self):
+        """parsing additional datetimes by pandas"""
+        date = ["1/22/20", "1/23/20", "1/24/20"]
```

Comment:
How about when dates are 1/1/20? How do you know, which is day and which is month? M/D/Y is not universal.

Reply:
if data is not in iso8601 format then month first is assumed. Trying to parse dates without specifying the format is always tricky. Perhaps an additional field to specify d-m-y order would be the solution. However, that would probably be in a new pr. I would really like to merge this since I need it to pares such dates.